pg_dump.go

```diff
@@ -34,8 +34,11 @@ type Dump struct {
 	fileName string
 }
 
-func NewDump(pg *Postgres) *Dump {
-	return &Dump{Options: PGDumpStdOpts, Postgres: pg}
+func NewDump(pg *Postgres) (*Dump, error) {
+	if !CommandExist(PGDumpCmd) {
+		return nil, fmt.Errorf("pg_dump command not found")
```

Comment:
this should be declared separatly:
```go
ErrCommandNotFound = errors.New("command not found")
```
and returned with
```go
return ErrCommandNotFound
```

Or
```go
struct ErrCommandNotFound{
   Command string
}

func (e ErrCommandNotFound) Error string() {
  return fmt.Sprintf("command not found: %s", e.Command)
}
```
and returned with:
```go
return ErrCommandNotFound{Command: PgDumpCmd}
```